test/extensions/stats_sinks/hystrix/hystrix_test.cc

```diff
@@ -209,6 +212,14 @@ class HystrixSinkTest : public testing::Test {
     return cluster_message_map;
   }
 
+  histogram_t* makeHistogram(const std::vector<uint64_t>& values) {
+    histogram_t* histogram = hist_alloc(); // The histogram needs to be freed later
```

Comment:
Should we have a RAII wrapper for this?

Reply:
Yes; if you use `hist_alloc()` in other places in your source, I recommend the same pattern.

Reply:
I don't, but I am tempted to change it in the test I borrowed from, thread_local_store_test.